cfme/services/catalogs/catalog_item.py

```diff
@@ -265,28 +286,39 @@ def delete(self):
         view.flash.assert_success_message('The selected Catalog Item was deleted')
 
     def add_button_group(self):
+        button_name = fauxfactory.gen_alpha()
         view = navigate_to(self, 'AddButtonGroup')
         view.fill({'btn_group_text': "group_text",
-                   'btn_group_hvr_text': "descr",
-                   'btn_image': "Button Image 1"})
+                   'btn_group_hvr_text': button_name,
+                   'btn_image': "Button Image 1" if self.appliance.version < '5.9' else 'broom'})
         view.add_button.click()
         view = self.create_view(DetailsCatalogItemView)
         assert view.is_displayed
-        view.flash.assert_success_message('Buttons Group "descr" was added')
+        return button_name
```

Comment:
**Required**
I would like to keep a call to view.flash.assert_no_error() here - as you did below in add_button.